.github/workflows/publish_docker.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     outputs:
       # Add a version to this list to automatically build Dockerfiles from it for the next releases
       default_cuda_images: '["nvidia/cuda:11.8.0-base-ubuntu22.04"]'
```

Comment:
Maybe not directly related to dropping py3.8 but rather supporting py3.11 and the [python-slim switch](https://github.com/Substra/substrafl/pull/197) but should we not rework this workflow in consequence?